lib/seattleflu/db/cli/command/clinical.py

```diff
@@ -230,12 +233,12 @@ def insert_clinical(df: pd.DataFrame, cursor):
 def print_problem_barcodes(problem_barcodes: pd.DataFrame, output: str):
     """
     Given a pandas DataFrame of *problem_barcodes*, writes the data to
-    stdout unless a filename *output* is given.
+    the log unless a filename *output* is given.
     """
     if output:
         problem_barcodes.to_csv(output, index=False)
     else:
-        print(problem_barcodes.to_csv(index=False))
+        LOG.info(problem_barcodes.to_csv(index=False))
```

Comment:
++for outputting to the log. :-) This spits out the CSV as a one honkin' log message, though, and I wonder if it would be nicer to output each row of `problem_barcodes` as a separate, English log message?

I would also suggest the use of `LOG.warning` instead of `info` to imply some greater amount of attention.